service/controller/resource/deployment/releasecomponents.go

```diff
@@ -0,0 +1,30 @@
+package deployment
+
+import (
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+
+	releasev1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/release/v1alpha1"
+)
+
+var keyComponents = []string{"calico", "containerlinux", "etcd", "kubernetes"}
```

Comment:
We usually have been calling these the "core components". Might be worth renaming this and other derivative names to keep consistency.

Reply:
Updated 👍